BrowserKit/Sources/SiteImageView/FaviconURLProcessing/URLFetcher/HTMLDataRequest.swift

```diff
@@ -14,12 +14,17 @@ struct DefaultHTMLDataRequest: HTMLDataRequest {
 
         // swiftlint:disable line_length
         static let userAgent = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Safari/605.1.15"
+        static let accept = "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8"
```

Comment:
curious where do these values come from `q=0.9,*/*;q=0.8`?

Reply:
Check out the [Accept documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept) for notes on q-factor weighting. :) 

I actually just borrowed these exact weightings from what the Firefox desktop browser was sending. Because if I did a curl command with no headers (`curl --location --request GET --head 'https://crates.io/search?q=query'`) you only get a 404 and no response body. But as soon as you add the Accept header (`curl --location --request GET 'https://crates.io/search?q=%7BsearchTerms%7D' --header 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8'`) you will get a response. It's something with how crates.io set up their server but it's not unreasonable for us to include `Accept` in the headers since we are just trying to get HTML documents in this scenario.

Reply:
nice! thanks I'll check out that doc! I see the example uses the same params for that weighting as well.